### PR TITLE
Updating the final project a bit

### PR DESCRIPTION
**Pull request by carriewright11**
- Made a bonus about functions
- Added more detail about requirements in the example

### CODE REVIEW

**Comment by carriewright11** [comment]
Any thoughts about the final project @cansavvy  and @avahoffman ?

File: ProjectExample/Final_Project_Example.Rmd [hunk 1]
```diff
@@ -133,39 +159,87 @@ ggplot(data = int_Country,
...
+This tests if there is a colum name that is the same as what is supplied as `aspect`. This then gets pulled from each dataset for the t test.
+
+There appears to be no significant difference according to the p-value for `Social Support` or `Healthy life expectancy`, as they are less than 0.5.
+
+Disclaimer - this was intended to demonstrate how one might perform such an analysis. We are not claiming that there is a difference or is not a difference in social support or healthy life expectancy between the countries in these two regions. 
+
+## How this might get evaluated
+According to the guidelines: 
+
+1. The student described where the data came from including how the instructor could access the data.
```
Comment (avahoffman) [inline ProjectExample/Final_Project_Example.Rmd]: Maybe write this out as 1/1 or +1 point?

**Review by avahoffman — APPROVED** [review]
Looks good! I just have one question about scoring the points in the example.

File: ProjectExample/Final_Project_Example.Rmd [hunk 2]
```diff
@@ -117,8 +117,34 @@ ggplot(data = top5,
            y = Happiness_score )) +
     geom_point()
 ```
+I felt that this plot could be improved. So first I used I made `Country_name` a factor and then reordered it by the `Happiness_score` variable. 
+
+```{r}
+
+top5 <- top5 %>% mutate(Country_name = as_factor(Country_name))
```
Comment (cansavvy) [inline ProjectExample/Final_Project_Example.Rmd]:
```suggestion
top5 <- top5 %>% mutate(Country_name = as_factor(Country_name), 
    Country_name = fct_reorder(Country_name, Happiness_score))
```
You can combine these in one step.

File: ProjectExample/Final_Project_Example.Rmd [hunk 3]
```diff
@@ -117,8 +117,34 @@ ggplot(data = top5,
...
-I then decided to make a plot of a few countries of interest:
+
+
+I then decided to make a plot of a few countries of interest. Since I again wanted to have countries ordered by the `Happiness_score`, I figured I might look at additional countries, so I decided to use the full dataset `Happiness`. 
+
+```{r}
+
+
+Happiness <- Happiness %>%
+  mutate(Country_name = as_factor(Country_name)) %>%
```
Comment (cansavvy) [inline ProjectExample/Final_Project_Example.Rmd]: Same comment here.

File: ProjectExample/Final_Project_Example.Rmd [hunk 4]
```diff
@@ -133,39 +159,87 @@ ggplot(data = int_Country,
...
+test_reg_diff(aspect = c("Social support"))
+test_reg_diff(aspect = c("Healthy life expectancy"))
+```
+
+This tests if there is a colum name that is the same as what is supplied as `aspect`. This then gets pulled from each dataset for the t test.
+
+There appears to be no significant difference according to the p-value for `Social Support` or `Healthy life expectancy`, as they are less than 0.5.
+
+Disclaimer - this was intended to demonstrate how one might perform such an analysis. We are not claiming that there is a difference or is not a difference in social support or healthy life expectancy between the countries in these two regions. 
+
```
Comment (cansavvy) [inline ProjectExample/Final_Project_Example.Rmd:231]: Add a `sessionInfo()`!

**Review by cansavvy — APPROVED** [review]
A few comments! But seems good!

File: ProjectExample/Final_Project_Example.Rmd (hunk 1 above)
Comment (carriewright11) [reply ProjectExample/Final_Project_Example.Rmd]: great idea!

File: ProjectExample/Final_Project_Example.Rmd (hunk 2 above)
Comment (carriewright11) [reply ProjectExample/Final_Project_Example.Rmd]: yeah I did this to make it easier to understand - but indeed I think they can handle it so I will change it

File: ProjectExample/Final_Project_Example.Rmd (hunk 3 above)
Comment (carriewright11) [reply ProjectExample/Final_Project_Example.Rmd]: same as above :P

File: ProjectExample/Final_Project_Example.Rmd (hunk 4 above)
Comment (carriewright11) [reply ProjectExample/Final_Project_Example.Rmd:231]: of course! thanks!

**Pull request merged**